manifest.json

```diff
@@ -23,34 +23,21 @@
         "idle",
         "notifications",
         "storage",
-        "tabs",
         "unlimitedStorage",
+        "webNavigation",
         "webRequest",
-        "alarms",
-        "telemetry",
+        "webRequestBlocking",
         "history"
     ],
 
-    "telemetry": {
-        "pioneer_id": true,
-        "study_name": "citp-news-disinfo",
-        "ping_type": "pioneer-study",
-        "schemaNamespace": "pioneer-citp-news-disinfo",
-        "public_key": {
-            "id": "citp-news-disinfo",
-            "key": {
-                "crv": "P-256",
-                "kty": "EC",
-                "x": "LDByX3lSRSU624OfR9EMO3So_0uRt2sNCVzPdQUKbrY",
-                "y": "4Qu2FsVM8834l0GJG2ZA0JyJlX5Oe83jV54PZNyCSCA"
-            }
-        }
-    },
+    "content_security_policy":
+        "script-src 'self' 'sha256-csyiOLMfXk2f5pU99mqYFyshgnVYbdp6o9bnQ9hntPA=' 'sha256-nYNRfLKTaKqgi4+CK/mcp9hdSsmD8F17GWuo+vQGfqU='; object-src 'self'; worker-src 'self';",
+
 
     "browser_specific_settings": {
         "gecko": {
-            "id": "news.study@princeton.edu",
-            "strict_min_version": "60.0"
+            "id": "rally.news.study@princeton.edu",
```

Comment:
The add-ons team has asked us to change these to end with `@rally.mozilla.org`, could we use `princeton-news-study@rally.mozilla.org`?